packages/modelence/src/client/config.ts

```diff
@@ -0,0 +1,37 @@
+declare global {
+  var modelenceConfig: ModelenceConfig;
+}
+
+function getGlobalObject() {
+  if (typeof global !== 'undefined') {
+    return global;
+  }
+  if (typeof window !== 'undefined') {
+    return window;
+  }
+  if (typeof self !== 'undefined') {
+    return self;
+  }
+  return {
+    modelenceConfig: {} as ModelenceConfig,
+  };
```

Comment:
Shouldn't we just throw an error in this case? If we're returning a new empty object each time, this is probably not going to work as expected anyway. And this is client only code, so this should never be the case.